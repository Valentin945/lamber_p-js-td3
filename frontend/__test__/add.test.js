import React from 'react';
import Home from '../src/scenes/home.jsx';
import renderer from 'react-test-renderer';
import "isomorphic-fetch"

test('Link changes the class when hovered', () => {
  const component = renderer.create(
    <Home />
  );
  let tree = component.toJSON();
  expect(tree).toMatchSnapshot();
});