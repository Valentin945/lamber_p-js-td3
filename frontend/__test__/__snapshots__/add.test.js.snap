// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Link changes the class when hovered 1`] = `
<div>
  <div
    style={
      Object {
        "display": "flex",
        "justifyContent": "center",
      }
    }
  >
    <button
      className="custom-button"
      onClick={[Function]}
    >
      reload
    </button>
  </div>
</div>
`;
